e2e/libs/utility/utility.py

```diff
@@ -187,3 +187,12 @@ def get_name_suffix(*args):
         if arg:
             suffix += f"-{arg}"
     return suffix
+
+
+def get_backupstores():
```

Comment:
This looks like it belongs in the backupstore lib.

Reply:
Refined. Thank you!